src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.cs

```diff
@@ -2208,8 +2208,8 @@ private ToolStripItem GetNextItemHorizontal(ToolStripItem start, bool forward)
                 return null;
             }
 
-            Debug.WriteLineIf(s_selectionDebug.TraceVerbose && (current != -1), "[SelectDBG GetNextToolStripItem] Last selected item was " + ((current != -1) ? DisplayedItems[current].Text : ""));
-            Debug.WriteLineIf(s_selectionDebug.TraceVerbose && (current == -1), "[SelectDBG GetNextToolStripItem] Last selected item was null");
+            Debug.WriteLineIf(s_selectionDebug!.TraceVerbose && (current != -1), $"[SelectDBG GetNextToolStripItem] Last selected item was {((current != -1) ? DisplayedItems[current].Text : string.Empty)}");
```

Comment:
I don't think we need to do another check for `current != -1` in the string. We are already checking that in the first param of the `Debug.WriteLineIf`.
```suggestion
            Debug.WriteLineIf(s_selectionDebug!.TraceVerbose && (current != -1), $"[SelectDBG GetNextToolStripItem] Last selected item was {DisplayedItems[current].Text}");
```